did/base.py

```diff
@@ -150,6 +150,22 @@ def width(self):
         except (NoOptionError, NoSectionError):
             return MAX_WIDTH
 
+    @property
+    def border_char(self):
+        """ Border character to use for the report """
+        try:
+            return self.parser.get("general", "border_char")
+        except (NoOptionError, NoSectionError):
+            return "~"
+
+    @property
+    def border_count(self):
```

Comment:
What about syncing the separator count with the report `width`? It would nicely go together when the line width is increased. Except for `width=0` which is used for unlimited line width.

Reply:
I implemented these changes for a friend who specifically didn't want it to match the screen width, so I'd have to disagree with this one.

Reply:
Ok, I see, understood.